### PR TITLE
statistics: update schema and config

### PR DESCRIPTION
**Pull request by Garfield-fr**
* Updates jsonschema and mappings.
* Adds sort on the name field.
* Adds aggregation on the category field.

### CODE REVIEW

File: rero_ils/modules/stats_cfg/jsonschemas/stats_cfg/stats_cfg-v0.0.1.json [hunk 1]
```diff
@@ -0,0 +1,1061 @@
...
+                  }
+                }
+              ]
+            }
+          }
+        }
+      ]
+    }
+  },
+  "definitions": {
```
Comment (zannkukai) [inline rero_ils/modules/stats_cfg/jsonschemas/stats_cfg/stats_cfg-v0.0.1.json:1024]: very cosmetic : it seems convetions to place "definitions" before use it ; place `definitions` before main `properties`

**Review by zannkukai — APPROVED** [review]
I will be more difficult to add a new configuration into the JSON schema now 😅

**Review by vgranata — APPROVED** [review]

File: rero_ils/modules/stats_cfg/jsonschemas/stats_cfg/stats_cfg-v0.0.1.json [hunk 2]
```diff
@@ -0,0 +1,1061 @@
...
+          "required": [
+            "indicator"
+          ],
+          "title": "Catalogue",
+          "properties": {
+            "type": {
+              "title": "Type",
+              "type": "string",
+              "const": "catalogue",
+              "default": "catalogue",
```
[the remark this answers is not in the document]
Comment (Garfield-fr) [reply rero_ils/modules/stats_cfg/jsonschemas/stats_cfg/stats_cfg-v0.0.1.json:120]: I looked into other resources. We've always defined it that way.

File: rero_ils/modules/stats_cfg/jsonschemas/stats_cfg/stats_cfg-v0.0.1.json (hunk 1 above)
Comment (Garfield-fr) [reply rero_ils/modules/stats_cfg/jsonschemas/stats_cfg/stats_cfg-v0.0.1.json:1024]: In the other schemes, the definition is always at the bottom of the file.

File: rero_ils/modules/stats_cfg/jsonschemas/stats_cfg/stats_cfg-v0.0.1.json [hunk 3]
```diff
@@ -0,0 +1,1061 @@
...
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "Statistics configuration",
+  "additionalProperties": false,
+  "propertiesOrder": [
+    "name",
+    "description",
+    "frequency",
+    "category",
+    "active"
```
[the remark this answers is not in the document]
Comment (Garfield-fr) [reply rero_ils/modules/stats_cfg/jsonschemas/stats_cfg/stats_cfg-v0.0.1.json:11]: 👍🏻

**Pull request merged**